clar2wasm/src/standard/standard.wat

```diff
@@ -212,7 +213,7 @@
         (local.set $b3 (i32.wrap_i64 (i64.shr_u (local.get $b_hi) (i64.const 32))))
 
         ;; Do long multiplication over the chunks
-        ;; Result = a0b0 + 
+        ;; Result = a0b0 +
```

Comment:
Is there a good wat formatter that you're using?

Reply:
I'm just always purging trailing newlines on save.

Reply:
I believe this is just a trailing of end-of-lines whitespaces. We should find a way to enforce that.